main.tf

```diff
@@ -338,3 +361,54 @@ module "wandb" {
     module.gke_app
   ]
 }
+
+# proxy-only subnet
+resource "google_compute_subnetwork" "proxy" {
+  name          = "${var.namespace}-proxy-subnet"
+  ip_cidr_range = var.ilb_proxynetwork_cidr
+  purpose       = "REGIONAL_MANAGED_PROXY"
+  role          = "ACTIVE"
+  network       = local.network.id
+}
+
+## In order to support private link required min version 0.13.0 of operator-wandb chart
+
+module "sleep" {
+  source = "matti/resource/shell"
+  version = "1.5.0"
+
+  environment = {
+    TIME = timestamp()
+  }
+  command              = "sleep 450; date +%s"
+  command_when_destroy = "sleep 450"
+  trigger              = timestamp()
+  working_dir          = "/tmp"
+
+  depends = [
+    module.wandb
+  ]
+}
+
+data "google_compute_forwarding_rules" "my_forwarding_rules" {
```

Comment:
Should probably rename this

Reply:
data "google_compute_forwarding_rules" "all" {}